functions/quiz.js

```diff
@@ -1,9 +1,32 @@
-import * as firebase from 'firebase';
-import 'firebase/firestore';
+import Axios from 'axios';
 import base64 from 'react-native-base64';
 import { shuffleArray } from './util';
 
-export const transformAPIData = (data) => {
+export const getTriviaData = async (catId, diff) => {
+  let triviaData;
+  await Axios.get('https://opentdb.com/api_token.php?command=request')
+    .then(async (res) => {
+      if (res.data.response_code === 0) {
+        await Axios.get(
+          `https://opentdb.com/api.php?amount=10&encode=base64&category=${catId}&difficulty=${diff}&token=${res.data.token}`,
+        )
+          .then(async (response) => {
+            if (response.data.results.length === 10) {
+              triviaData = transformAPIData(response.data.results);
+            }
```

Comment:
Only storing categories with redux. When creating the quiz I just make the callout and pass the data in navigation. Also just getting a new token everytime it makes a callout - probably a better way handle the refresh token but can work it out later.